upload_settings_dialog.py

```diff
@@ -62,23 +62,16 @@ def __init__(self, upload_size, iface, project_definition):
         else:
             self.ui.title_le.setText(DEFAULTS['ISO19115_TITLE'])
 
-        # if no field is selected, we should not allow uploading
-        self.zone_label_field_is_specified = False
-        reload_attrib_cbx(
-            self.ui.zone_label_field_cbx, iface.activeLayer(), True)
-        # pre-select the field if it's specified in the project definition
-        if 'zone_label_field' in self.project_definition:
-            zone_label_idx = self.ui.zone_label_field_cbx.findText(
-                self.project_definition['zone_label_field'])
-            if zone_label_idx != -1:
-                self.ui.zone_label_field_cbx.setCurrentIndex(zone_label_idx)
-                self.zone_label_field_is_specified = True
```

Comment:
Same as above. We need to keep the selection of the zone_label_field, pre-populating the combobox if the project definition already contains such information